djautotask/migrations/0087_auto_20201221_1526.py

```diff
@@ -0,0 +1,57 @@
+# Generated by Django 3.1.2 on 2020-12-21 15:26
```

Comment:
It would be good to delete these 2 migration files and make a single new migration for the changes. As it currently is, the first migration adds a number of fields and the next migration removes some that were just added.

Reply:
Done. Thanks~ :+1: